src/helm/benchmark/presentation/schema.py

```diff
@@ -12,7 +12,9 @@
 
 
 SCHEMA_YAML_PACKAGE: str = "helm.benchmark.static"
-SCHEMA_YAML_FILENAME: str = "schema.yaml"
+
+# TODO: add heim, vhelm, etc.
+SCHEMA_CLASSIC_YAML_FILENAME: str = "schema_classic.yaml"
```

Comment:
Add TODO to change `SCHEMA_YAML_PACKAGE` from `helm.benchmark.static` to `helm.benchmark.config`

No action needed: We should consider breaking up the `schema.yaml` file into `adapters.yaml`, `metrics.yaml`, `perturbations.yaml` etc. The term "schema" is very overloaded.

Reply:
Yeah, I was thinking about doing that, but keeping one file does have some advantages.  Should discuss this some more.